input/omnia_config.yml

```diff
@@ -57,7 +57,6 @@ k8s_cni: "calico"
 # Mandatory Field
```

Comment:
You should also expose `supplementary_addresses_in_ssl_keys` from Kubespray to be able to add virtual IPs to the TLS certificate.